Source/WebCore/html/canvas/GPUCanvasContextCocoa.mm

```diff
@@ -178,13 +178,20 @@ static int getCanvasHeight(const GPUCanvasContext::CanvasType& canvas)
 
 void GPUCanvasContextCocoa::prepareForDisplayWithPaint()
 {
-    // FIXME: https://bugs.webkit.org/show_bug.cgi?id=233622 - Support offscreen canvas with WebGPU
     prepareForDisplay();
 }
 
 void GPUCanvasContextCocoa::paintRenderingResultsToCanvas()
 {
-    // FIXME: https://bugs.webkit.org/show_bug.cgi?id=233622 - Support offscreen canvas with WebGPU
+    bool canvasContainsDisplayBuffer = false;
+    if (!canvasContainsDisplayBuffer) {
```

Comment:
Why is this idiom being used? `canvasContainsDisplayBuffer` is always false and never updated.

Reply:
Whups that was from copying the WebGL implementation and not cleaning it up, I will remove that, thanks for catching